core/benches/blocks/common.rs

```diff
@@ -187,12 +187,12 @@ pub fn build_wsv(
     );
     let mut wsv = WorldStateView::new(World::with([domain], UniqueVec::new()), kura, query_handle);
     wsv.config.transaction_limits = TransactionLimits::new(u64::MAX, u64::MAX);
-    wsv.config.wasm_runtime_config.fuel_limit = u64::MAX;
-    wsv.config.wasm_runtime_config.max_memory = u32::MAX;
+    wsv.config.wasm_runtime.fuel_limit = u64::MAX;
+    wsv.config.wasm_runtime.max_memory = u32::MAX.into();
```

Comment:
what is the meaning of this? we are only going to allow `u64` in config?

Reply:
I made a newtype for byte sizes - `ByteSize<T>`. This is a simple conversion into it.
I wanted to discuss the newtype itself, but anyway I think it is better to write `ByteSize(u32::MAX)` here to avoid confusion.

Reply:
Why do we need this newtype?

Reply:
> Why do we need this newtype?

- In the config: allows (later) parsing bytes from human-readable values like `2mb`
- In the rest of the codebase: gives a stronger type, clarifies "blind" numeric type

Reply:
Anyways, I made it just `u32` again (but with `_bytes` suffix). I will propose to introduce a normal `Bytes` newtype in a separate PR.